features/fixtures/mazerunner/cxx-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/CXXThrowSomethingOutsideReleaseStagesScenario.java

```diff
@@ -17,7 +15,11 @@
 
     public native void crash(int num);
 
-    public CXXThrowSomethingOutsideReleaseStagesScenario(@NonNull Configuration config, @NonNull Context context) {
+    /**
```

Comment:
Do we really want an empty JavaDoc?

Reply:
Added comment to this Javadoc - this is to suppress a checkstyle rule that public methods over 1 line long must have a comment.